### DISCUSSION TITLE
Help to figure out if the cam is bricked

### DISCUSSION THREAD

**Discussion by troyane** [roleoroleo/yi-hack-Allwinner-v2#708] [Q&A]
I've got 2 cams exactly the same as described here https://github.com/roleoroleo/yi-hack-Allwinner-v2/discussions/653

Cams were without SD cards and did nothing when I turned them on, except for the lighting backlight.

I got the same cam from this batch and successfully hacked it. It was `b091qp` and it works flawlessly.

How can I ensure that cams are not bricked and bring them to life?

Thanks!

**Comment by roleoroleo** [comment]
Could you explain me better?

**Comment by troyane** [comment]
Sure, @roleoroleo 
I got 2 used visually equal cameras without SD cards. They won't boot up when I plug power cord. I can see only IR lights.
Tried to prepare SD card with hack, but seems like nothing happens: SD card content remains the same. 

If I'm going to "unbrick" it I need few files `mtdblockX.bin`, but I got none of them. How can I proceed?

**Comment by roleoroleo** [comment]
I can share with you a backup of the model you mentioned in the 1st post.
Unlike all other models, the suffix b091qp is shared between different manufacturers.
So, you can't be sure you're using the correct mtdblock if it's not yours.
If you want to try...

**Comment by roleoroleo** [comment]
Or...
If the other cam is working and is identical, you can use its partiions.
